packages/hardhat/hardhat.config.ts

```diff
@@ -1,4 +1,5 @@
 import { HardhatUserConfig } from "hardhat/config";
+import { config as dotenvConfig } from "dotenv";
```

Comment:
For these hardhat config changes, would you mind posting them to: https://github.com/FIL-Builders/fil-frame/pull/24?

And revert all changes here?

Better for organization; and also you can take a look at the feedback that I gave previously.

Removing the un-needed networks is a great move.

Reply:
I can do that but the problem was that without this I couldn't test on calibration the deployment => verification as the env was not readable.

Reply:
@xBalbinus Moved hardhat config there

Reply:
Oh for sure, that's totally fine